src/commands/moderation/kick.ts

```diff
@@ -38,50 +38,64 @@ export default class Kick {
 
     const dmChannel = (await user.createDM().catch(() => null)) as DMChannel | null;;
     const kickEmbed = new EmbedBuilder()
+      .setAuthor({ name: `• ${user.username}`, iconURL: user.displayAvatarURL() })
       .setTitle(`✅ • Kicked <@${user.id}>`)
       .setDescription([
         `**Moderator**: <@${interaction.user.id}>`,
         `**Reason**: ${reason ?? "No reason provided"}`
       ].join("\n"))
       .setThumbnail(user.displayAvatarURL())
-      .setAuthor({ name: `• ${user.username}`, iconURL: user.displayAvatarURL() })
       .setFooter({ text: `User ID: ${user.id}` })
       .setColor(genColor(100));
 
     const embedDM = new EmbedBuilder()
+      .setAuthor({ name: `• ${user.username}`, iconURL: user.displayAvatarURL() })
       .setTitle(`👢 • You were kicked`)
```

Comment:
```suggestion
      .setTitle(`🥾 • You were kicked`)
```
This one probably fits better I think

Reply:
next commit has the emoji updated